pkg/cloud/gen.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2023 Google LLC
+Copyright 2024 Google LLC
```

Comment:
This should go in a different commit.

Reply:
it is auto generated in `make` - I removed this form commit